CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 0.1.1 - 2020-06-10
+
+### Added
```

Comment:
What about "Open Repo in GitHub"?

Feels like that plus the functions thing is two good features to me. Aka worth 0.2.0

Reply:
![1](https://media.giphy.com/media/eNq8r4XARG9dcRYLA8/giphy.gif)

why you gotta be so negative 😂